vercel-api/.github/workflows/ci.yml

```
@@ -0,0 +1,393 @@
+    - uses: actions/checkout@v4
+    
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+        cache: 'npm'
+        cache-dependency-path: vercel-api/package-lock.json
+    
+    - name: Install Vercel CLI
+      run: npm install --global vercel@latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)